src/bin/interactive.rs

```diff
@@ -258,11 +259,26 @@ pub(crate) fn load_wallet(
     // display main menu
     let wallet = match menu_wallet(wallet_found) {
         MainMenu::Load(path) => {
-            let pwd = prompt::request_auth(
-                "Please enter you wallet's password",
-                password,
-            );
-            Wallet::from_file(WalletFile { path, pwd })?
+            let mut attempt = 1;
+            loop {
+                let pwd = prompt::request_auth(
+                    "Please enter you wallet's password",
+                    password,
+                );
+                match Wallet::from_file(WalletFile {
+                    path: path.clone(),
+                    pwd,
+                }) {
+                    Ok(res) => break res,
+                    Err(err) if attempt > 2 => {
+                        return Err(err)?;
```

Comment:
From the issue #46
> If the password is 3 times wrong, it should go directly to the recovery process

Should the wallet be removed if password is wrong 3 times in a row?
If so, feel free to handle it in a different PR in order to not block this one

Reply:
That's a good point, think that makes sense will open a new pr

Reply:
I would strongly suggest to open a new issue first – in any case, we should have PRs without issue – and use such issue to discuss if  and how we should implement this feature.